lms/templates/index.html

```diff
@@ -43,13 +47,13 @@
             <div class="secondary-actions">
               <div class="social-sharing">
                 <div class="sharing-message">${_("Stay up to date with all {platform_name} has to offer!").format(platform_name=settings.PLATFORM_NAME)}</div>
-                <a href="https://twitter.com/edXOnline" class="share">
+                <a href="${settings.PLATFORM_TWITTER_URL}" class="share">
```

Comment:
We didn't want to add conditionalization of these images based on whether the URLs exist?